03-Installation.md

```diff
@@ -0,0 +1,33 @@
+---
+title:     Installing TGMPA
+menutitle: Installation
+anchor:    installation
+permalink: /installation/
+---
+
+### {{page.title}}
+
+Installing the **{{site.tgmpa.name}}** library is easy. Follow the instructions below:
+
+1. [Download] the latest release.
+2. Drop the class file somewhere in your theme or plugin hierarchy.
+3. Add a `require_once` call within `functions.php` (or other theme or plugin file) referencing the class file.
+4. Create a function, hooked to `tgmpa_register`, that registers the plugin and configurations.
+
+
+For steps 3 and 4, it is **_strongly recommended_** you view, copy and paste the contents of the `example.php` file which is included in the release package and amend to suit. The `example.php` file is a model for how you should include the class in your theme.
```

Comment:
@grappler I could, but am a bit wary - I'd prefer if people use the file included in their download as that's the one belonging to the version they have.